File: C#/Equipo.cs
using System;

public class Equipo {
	private String nombre;
	private int id;
	private ScrumMaster scrumMaster;
    private Integrante integrantes;

	public String GetNombre() {
		return this.nombre;
	}
	public void SetNombre(ref String nombre) {
		this.nombre = nombre;
	}
	public int GetId() {
		return this.id;
	}
	public void SetId(ref int id) {
		this.id = id;
	}
	public ScrumMaster GetScrumMaster() {
		return this.scrumMaster;
	}
	public void SetScrumMaster(ref ScrumMaster scrumMaster) {
		this.scrumMaster = scrumMaster;
	}
    public Integrante getIntegrantes() {
		return integrantes;
	}

	public void setIntegrantes(Integrante integrantes) {
		this.integrantes = integrantes;
	}

    public void AddIntegrate(ref Integrante integrante) {
	}
	public Equipo(int id, String nombre, ScrumMaster scrumMaster, Integrante integrantes) {
		this.id = id;
        this.nombre = nombre;
        this.scrumMaster = scrumMaster;
        this.integrantes = integrantes;
	}
    public override String ToString() {
		return base.ToString() + "Id: "+ id.ToString() + "\n" + "Nombre : " + nombre.ToString() +"\n"
        + "Encargado : " + scrumMaster.ToString() +"\n" + "Integrantes : " + integrantes.ToString() +"\n";
	}

}

File: Equipo.java
import java.util.Vector;

public class Equipo {
	private String nombre;
	private int id;
	private ScrumMaster scrumMaster;
	public Vector<Integrante> integrantes = new Vector<Integrante>();

	public String getNombre() {
		return this.nombre;
	}

	public void setNombre(String aNombre) {
		this.nombre = aNombre;
	}

	public int getId() {
		return this.id;
	}

	public void setId(int aId) {
		this.id = aId;
	}

	public void addIntegrate(Integrante aIntegrante) {
		throw new UnsupportedOperationException();
	}

	public void toString() {
		throw new UnsupportedOperationException();
	}

	public ScrumMaster getScrumMaster() {
		return this.scrumMaster;
	}

	public void setScrumMaster(ScrumMaster aScrumMaster) {
		this.scrumMaster = aScrumMaster;
	}

	public Equipo(int aId, String aNombre, ScrumMaster aSm) {
		throw new UnsupportedOperationException();
	}
}